sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -118,6 +118,28 @@ def test_sanity_check_pls_regression():
     assert_array_almost_equal(y_loadings_sign_flip, y_weights_sign_flip)
 
 
+@pytest.mark.parametrize("Estimator", (PLSRegression, PLSCanonical, CCA))
```

Comment:
You could parametrize the test by passing `X` to be 10 samples and then a single sample